### PR TITLE
Initial Docs

### PR DESCRIPTION
**Pull request by somebody32**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
 
-Documentation, as always, is not yet ready. See `examples` folder.
+Creates reducers and listeners for your backbone collections and fires action creators on every collection change.
+
+### Why?
+
+* You can start migrating your apps from backbone to react+redux in no time (big blogpost about our experience is coming)
+* No need to worry about migrated/legacy parts of your app being out of sync, because both are using single source of truth
+* No boilerplate
+* You can hide all new concepts like `reducers`, `stores` and `purity` from other devs to avoid brain overloading.
```
Comment (masylum) [inline README.md]: add action creators too. Also, I would add: "Those are just implementation details to make React performant".

File: README.md [hunk 2]
```diff
@@ -1,6 +1,59 @@
...
+
```
Comment (masylum) [inline README.md:20]: I would also add: "REST enabled by default. Most React projects end up implementing an ad hoc, bug-ridden implementation of Backbone.Collection not only once but for each store."

File: README.md [hunk 3]
```diff
@@ -1,6 +1,59 @@
...
+import { devTools } from 'redux-devtools';
+import { syncCollection } from 'backbone-redux';
+
+//  Create you redux-store, include all middlewares you want.
+const finalCreateStore = compose(devTools())(createStore);
+const store = finalCreateStore(() => {}); // Store with just empty dumb reducer
+
+// Now just call auto-syncer from backbone-redux
+// Assuming you have app.todos Backbone collection
+syncCollection({todos: app.todos}, store);
```
Comment (masylum) [inline README.md]: should we import `app`? Maybe call it `collections` so its more self-explanatory

File: README.md [hunk 4]
```diff
@@ -1,6 +1,59 @@
...
+    byId: {
+      1: {id: 1, ...},
+      2: {id: 2, ...}
+    }
+  }
+}
+```
+
+Every object in entities array is not a Backbone model, but a result of calling `toJSON` on it + field called `__optimistic_id` which is equal to model's `cid`;
+
```
Comment (masylum) [inline README.md:56]: I think people may be interested how to interact with the collections. I think visualizing the one-way flow should be inspiring

Comment (masylum) [inline README.md:56]: let me draw that for you

File: README.md (hunk 1 above)
Comment (somebody32) [inline README.md]: Not only performance, I'm still thinking that separating `server data` and `UI data` is a good idea. It just helps you to keep data flat and work with components without feeling that you have legacy, so as a result component composition and props _should be_ cleaner.

File: README.md (hunk 4 above)
Comment (masylum) [inline README.md:56]: https://www.lucidchart.com/invitations/accept/cc298cc1-3d85-47c3-ae14-9a313b9a2f6c

Comment (masylum) [inline README.md:56]: (lucid chart in meldium)

Comment (somebody32) [inline README.md:56]: fuu, done it with pseudo-asci

**Pull request merged**